### PR TITLE
Updated side banners and made them switchable with light and dark modes

### PR DESCRIPTION
**Pull request by danielyj98**

### CODE REVIEW

File: src/Components/Navigation/SidePhoto.js [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
-      <Box
-        mt="-16"
-        right={0}
-        zIndex={-2}
-        pos={"fixed"}
-        borderLeft={"1px"}
-        borderColor={"gray.700"}
-      >
-        <Image src="https://i.ibb.co/FqbjLw6/dark-banner-2.png"></Image>
+      <Box mt="-16" right={0} zIndex={-2} w="20%" pos={"fixed"} X>
```
Comment (evanfordebarden) [inline src/Components/Navigation/SidePhoto.js:7]:
```suggestion
      <Box mt="-16" right={0} zIndex={-2} w="20%" pos={"fixed"}  
      h="100%" 
      bgSize="100%" 
      bgImage={`url(${photo})`}
      backgroundRepeat="repeat"
      bgPosition="top">
      X // is this a typo?
```

this should make the image fill the right 20% of the page from top to bottom at all sizes.

File: src/Components/Navigation/SidePhoto.js [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
+        <Image src={photo}></Image>
```
Comment (evanfordebarden) [inline src/Components/Navigation/SidePhoto.js:8]:
```suggestion

```

suggestion above makes this line unecessary

**Review by evanfordebarden — CHANGES_REQUESTED** [review]
On a larger screen, the image was not filling the entire vertical height. Suggested changes should fix it!

File: src/Components/Navigation/SidePhoto.js (hunk 1 above)
Comment (danielyj98) [reply src/Components/Navigation/SidePhoto.js]: Nice catch, and great job on making it a background photo!

File: src/Components/Navigation/SidePhoto.js [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
-        borderColor={"gray.700"}
-      >
-        <Image src="https://i.ibb.co/FqbjLw6/dark-banner-2.png"></Image>
+      <Box mt="-16" right={0} zIndex={-2} w="20%" pos={"fixed"}  
+      h="100%" 
+      bgSize="100%" 
+      bgImage={`url(${photo})`}
+      backgroundRepeat="repeat"
+      bgPosition="top">
+      X // is this a typo?
```
Comment (evanfordebarden) [inline src/Components/Navigation/SidePhoto.js:13]: one more thing, let's delete this line? i accidentally moved it outside of the <Box> tag, but there was a seemingly random "X" in there to begin with.

```suggestion

```

Comment (danielyj98) [reply src/Components/Navigation/SidePhoto.js:13]: My bad- thought it was taken out in the revision, but absolutely

**Review by evanfordebarden — APPROVED** [review]
everything looks good!

**Pull request merged**